### DISCUSSION TITLE
Make ammo boxes combinable

### DISCUSSION THREAD

**Discussion by marpul358** [FakeFishGames/Barotrauma#8931] [Ideas and suggestions]
Ammo box A has 50% left. Ammo box B has 25% left. Would be nice to have the possibility to combine these so one would have 75% left and the other 0%. Wouldn't have to go into fights with nearly empty ammo boxes and wouldn't get stuck with the percentage based resources that you get when you deconstruct non-full ammo boxes.

**Comment by AshEdwardsDesign** [comment]
+1 this would be a nice QOL improvement.

**Comment by mygamingaccount** [comment]
Always have this mod in your list until this request is fulfilled 
https://steamcommunity.com/sharedfiles/filedetails/?id=2799051561
